web/src/pages/apd/activities/cost-allocation/CostAllocation/OtherFunding.js

```diff
@@ -72,12 +134,30 @@ const OtherFunding = ({
                 className: 'ds-h5'
               }}
             />
-            <DollarField
-              name={`ffy-${ffy}`}
-              label={`FFY ${ffy}`}
-              labelClassName="sr-only"
-              value={costAllocation[ffy].other || '0'}
-              onChange={setOther(ffy)}
+            <Controller
+              name={`costAllocation.${ffy}.other`}
+              control={control}
+              value={costAllocation[ffy]?.other || '0'}
+              render={({ 
+                field: { onChange, value, ...props }
+              }) => (
+                <DollarField
+                  {...props}
+                  value={value}
+                  label={`FFY ${ffy}`}
+                  labelClassName="sr-only"
+                  onChange={e => {
+                    setOtherFunding(activityIndex, ffy, value);;
+                    onChange(e);
+
+                    if (adminCheck) {
+                      trigger();
+                    }
+                  }}
+                  errorPlacement="bottom"
+                  errorMessage={errors?.costAllocation && errors?.costAllocation[ffy]?.other?.message}
```

Comment:
you can switch this to `errors?.costAllocation?.[ffy]?.other?.message` and remove the `errors?.costAllocation &&`

Reply:
it should be 
```
errorMessage={errors?.costAllocation?.[ffy]?.other?.message}
```